Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_and_SUNBURST_Hunting_and_Response_Playbook_6.yml

```diff
@@ -2297,7 +2299,8 @@ tasks:
     task:
       id: 19541626-dcaf-4f4a-89a2-45d193c85535
       version: -1
-      name: Block vulnerable SolarWinds Server Automatically?
+      name: Block vulnerable SolarWinds server automatically?
+      description: Checks if block the vulnerable SolarWinds server automatically according to playbook input
```

Comment:
description: Checks whether to block the vulnerable SolarWinds server automatically according to the value set for "AutoBlockSolarWindsServer" playbook input.